.github/workflows/deploy.yaml

```diff
@@ -64,7 +64,8 @@ jobs:
   mintlify_deployment:
     runs-on: depot-ubuntu-24.04-4
     needs:
-      - api_production_deployment
+      - detect_changes
+    if: needs.detect_changes.outputs.docs == 'true'
```

Comment:
Decoupling documentation deployment from API deployment so docs can be independent